ocaml/libs/xapi-stdext/lib/xapi-stdext-zerocheck/zerocheck_stub.c

```diff
@@ -31,7 +31,7 @@ value is_all_zeros(value string, value length)
 	for (i = len / 4; i > 0; i--)
 		if (*p++ != 0)
 			goto notallzero;
-	s = (unsigned char *) p;
+	s = (const char *) p;
```

Comment:
Not a regression but looking at the above code even `p` should be a `const` pointer. There's also the assumption that `sizeof(int) == 4` but I suppose code is not intended to be that portable, although a `uint32_t` pointer would maybe be better... or replace the `4` with `sizeof(int)` or `sizeof(*p)`.